changelog/Microsoft.AAD.Reporting.json

```
@@ -1,5 +1,31 @@
 {
   "changelog": [
+      {
+         "ChangeList": [
+           {
+            "Id": "32ae0a92-8eca-4f25-8552-7600bc9370f5",
+            "ApiChange": "Property",
+            "ChangedApiName": "isAdmin",
+            "ChangeType": "Addition",
+            "Description": "Added the `isAdmin` property to the [userRegistrationDetails](https://docs.microsoft.com/en-us/graph/api/resources/userRegistrationDetails?view=graph-rest-beta) resource.",
+            "Target": "userRegistrationDetails"
+          },
+          {
+            "Id": "32ae0a92-8eca-4f25-8552-7600bc9370f5",
+            "ApiChange": "Property",
+            "ChangedApiName": "userType",
+            "ChangeType": "Addition",
+            "Description": "Added the `userType` property to the [userRegistrationDetails](https://docs.microsoft.com/en-us/graph/api/resources/userRegistrationDetails?view=graph-rest-beta) resource.",
+            "Target": "userRegistrationDetails"
+          }
+        ],
+        "Id": "32ae0a92-8eca-4f25-8552-7600bc9370f5",
+        "Cloud": "Prod",
+        "Version": "beta",
+        "CreatedDateTime": "2022-06-29T18:50:22.4220905Z",
+        "WorkloadArea": "Reports and audit",
```

Should map to TOC H2 otherwise changelog filter experience won't show it.
```suggestion
        "WorkloadArea": "Reports",
```